api/room/schema_query.py

```diff
@@ -53,6 +53,13 @@ class DailyEvents(graphene.ObjectType):
     events = graphene.List(CalendarEvent)
 
 
+class PaginatedEvents(graphene.ObjectType):
```

Comment:
Great work @verenceLola, Please provide the documentation for this class.

Reply:
This has been worked on @RachaelNantale .

Reply:
good work @verenceLola, you could also go ahead and add some field descriptions. something like 
`pages = graphene.Int(description="The pages number field of the events")`